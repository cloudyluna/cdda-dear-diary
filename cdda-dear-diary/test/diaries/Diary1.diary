Entry: 1/3, Year 1, Spring, day 61 08:02:25, 

Stats:
Strength: 14
Dexterity: 12
Intelligence: 12
Perception: 12
 
Skills:
computers: 1
electronics: 1
fabrication: 2
marksmanship: 2
mechanics: 1
survival: 2
devices: 1
health care: 2
athletics: 2
archery: 1
bashing weapons: 1
cutting weapons: 1
rifles: 3
piercing weapons: 1
vehicles: 3
social: 1
food handling: 1
tailoring: 1
applied science: 1

Proficiencies:
Lockpicking
Wound Care
Driving
Food Preparation
Garment Closures
Principles of Leatherworking
Pottery
Pottery Glazing

Mutations:
Open Perk Menu
Eye color: gray
Quick
Bookworm
Fast Reflexes
Sociable
Good Hearing
Outdoorsman
Masochist
Hair: brown, ponytail
Skin tone: light
Packmule
KÃ¼nstler
Fast Learner
Optimist
Pain Resistant
Night Vision
Pretty

Active missions:
Find some cosplay suits
Faction succession
 
THE WORLD HAS GONE NUTS.

LAWS ARE NO MORE.

I have to fend myself off from the monsters...and 
I think the humans are far scarier than those.

Who do I trust??


Wish me luck.



Entry: 2/3, Year 1, Spring, day 61 10:18:02, 2 hours, 15 minutes since last entry

Stats: 
Strength: 14 -> 25
 
Skills: 
marksmanship: 2 -> 3
survival: 2 -> 3
 
Gained Mutation: 
no skill rust
 
Kills: 
  16 Z zombies
   7 Z fat zombies
   1 Z decayed zombie
   5 z zombie children
   1 d Z-9
   1 Z tough zombie
   1 @ petrified person
   1 @ gawky guardian
 
Dear diary,

I just fought a swarm of zombies. Small and big. 
They were though but I managed.

I hope I can make it out.

I just need to fight.

I need to.

- Cat



Entry: 3/3, Year 1, Spring, day 61 12:10:30, 1 hour, 52 minutes since last entry

Lost Mutation: 
no skill rust
 
Kills: 
   2 Z zombies
 
Dear diary,

I found a little shack of some kind.

Along the way, I encountered a shopping cart. So 
my back will be relief of the loot burden.

I feel a bit better now. I hope this will last.

- Cat



