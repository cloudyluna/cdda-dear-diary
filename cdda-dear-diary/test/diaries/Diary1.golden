Right (MkDiary {totalIndexes = 3, entries = [MkEntry {index = 1, dateTime = MkDateTime {season = Spring, day = 61, year = 1, time = MkTime {hour = 8, minute = 2, second = 25}}, lastWrittenTime = Nothing, body = MkBody {stats = Just (Initiation (MkCharacterStats {strength = 14, dexterity = 12, intelligence = 12, perception = 12})), skills = Just (Initiation (MkCharacterSkills {computers = 1, electronics = 1, fabrication = 2, marksmanship = 2, mechanics = 1, survival = 2, devices = 1, healthCare = 2, athletics = 2, archery = 1, bashingWeapons = 1, cuttingWeapons = 1, rifles = 3, piercingWeapons = 1, vehicles = 3, social = 1, foodHandling = 1, tailoring = 1, appliedScience = 1})), proficiencies = Just (Initiation ["Lockpicking","Wound Care","Driving","Food Preparation","Garment Closures","Principles of Leatherworking","Pottery","Pottery Glazing"]), mutations = Just (Initiation ["Open Perk Menu","Eye color: gray","Quick","Bookworm","Fast Reflexes","Sociable","Good Hearing","Outdoorsman","Masochist","Hair: brown, ponytail","Skin tone: light","Packmule","K\252nstler","Fast Learner","Optimist","Pain Resistant","Night Vision","Pretty"]), activeMissions = Just (Initiation ["Find some cosplay suits","Faction succession"]), newMissions = Nothing, gainedMutation = Nothing, lostMutation = Nothing, kills = Nothing, npcKilled = Nothing, note = "THE WORLD HAS GONE NUTS.\n\nLAWS ARE NO MORE.\n\nI have to fend myself off from the monsters...and \nI think the humans are far scarier than those.\n\nWho do I trust??\n\n\nWish me luck.\n\n\n\n"}},MkEntry {index = 2, dateTime = MkDateTime {season = Spring, day = 61, year = 1, time = MkTime {hour = 10, minute = 18, second = 2}}, lastWrittenTime = Just (MkLastWrittenTime {day = Nothing, hour = Just 2, minute = 15}), body = MkBody {stats = Just (Update (fromList [("Strength",MkStatUpdate {from = 14, to = 25})])), skills = Just (Update (fromList [("marksmanship",MkStatUpdate {from = 2, to = 3}),("survival",MkStatUpdate {from = 2, to = 3})])), proficiencies = Nothing, mutations = Nothing, activeMissions = Nothing, newMissions = Nothing, gainedMutation = Just (Update ["no skill rust"]), lostMutation = Nothing, kills = Just (Update [MkCharacterKills {count = 16, symbol = "Z", name = "zombies"},MkCharacterKills {count = 7, symbol = "Z", name = "fat zombies"},MkCharacterKills {count = 1, symbol = "Z", name = "decayed zombie"},MkCharacterKills {count = 5, symbol = "z", name = "zombie children"},MkCharacterKills {count = 1, symbol = "d", name = "Z-9"},MkCharacterKills {count = 1, symbol = "Z", name = "tough zombie"},MkCharacterKills {count = 1, symbol = "@", name = "petrified person"},MkCharacterKills {count = 1, symbol = "@", name = "gawky guardian"}]), npcKilled = Nothing, note = "Dear diary,\n\nI just fought a swarm of zombies. Small and big. \nThey were though but I managed.\n\nI hope I can make it out.\n\nI just need to fight.\n\nI need to.\n\n- Cat\n\n\n\n"}},MkEntry {index = 3, dateTime = MkDateTime {season = Spring, day = 61, year = 1, time = MkTime {hour = 12, minute = 10, second = 30}}, lastWrittenTime = Just (MkLastWrittenTime {day = Nothing, hour = Just 1, minute = 52}), body = MkBody {stats = Nothing, skills = Nothing, proficiencies = Nothing, mutations = Nothing, activeMissions = Nothing, newMissions = Nothing, gainedMutation = Nothing, lostMutation = Just (Update ["no skill rust"]), kills = Just (Update [MkCharacterKills {count = 2, symbol = "Z", name = "zombies"}]), npcKilled = Nothing, note = "Dear diary,\n\nI found a little shack of some kind.\n\nAlong the way, I encountered a shopping cart. So \nmy back will be relief of the loot burden.\n\nI feel a bit better now. I hope this will last.\n\n- Cat\n\n\n\n"}}]})